package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "space-travelers-hub",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.8.3",
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.3.0",
+    "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
+    "bootstrap": "^5.2.0",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-bootstrap": "^2.4.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.0.2",
+    "react-router-dom": "^5.3.0",
+    "react-scripts": "5.0.1",
+    "redux-thunk": "^2.4.1",
+    "react-test-renderer": "^18.2.0",
```

Comment:
- [ ] It would be better if we add the redux-logger package to our package.json file, this will help us replay problems as if they happened in our own browser

- [ ] Also kindly fix the design so it aligns with the other items as in the screenshot below
![Screenshot from 2022-07-21 08-40-40](https://user-images.githubusercontent.com/46388113/180170542-7123f4d6-4461-47cb-8a96-35ce4e19b3d8.png)